lib/SIL/SILGlobalVariable.cpp

```diff
@@ -76,12 +76,13 @@ SILInstruction *SILGlobalVariable::getStaticInitializerValue() {
   return &StaticInitializerBlock.back();
 }
 
-bool SILGlobalVariable::isValidStaticInitializerInst(const SILInstruction *I) {
+bool SILGlobalVariable::isValidStaticInitializerInst(const SILInstruction *I,
+                                                     SILModule &M) {
   switch (I->getKind()) {
     case ValueKind::BuiltinInst: {
       auto *bi = cast<BuiltinInst>(I);
-      switch (bi->getBuiltinInfo().ID) {
-        case BuiltinValueKind::FPTrunc:
```

Comment:
What happens to the FPTrunc case here?

Reply:
I intentionally removed it. We don't need it because fptrunc can always be constant folded for initializers (because the argument must be a literal). Also, we never hit this case because it would have crashed anyway.